### PR TITLE
TECH-177: Activate Mentors in FlashCampus when Onboarding Complete in…

### PR DESCRIPTION
**Pull request by daniyusra**
Implement activation to be fired whenever mentor finishes their onboarding in Cal.com

### CODE REVIEW

File: packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts [hunk 1]
```diff
@@ -399,3 +404,41 @@ const handleUserMetadata = ({ ctx, input }: UpdateProfileOptions) => {
...
+  const { user } = ctx;
+
+  if (user.username != null) {
+    const email = user.email;
+
+    const defaultLink = `${process.env.NEXT_PUBLIC_WEBAPP_URL}/${user.username}/15min_meeting`;
+
+    const url = `${process.env.FLASHCAMPUS_LINK}/api/account/activate-mentor`;
+
+    console.log(defaultLink);
```
Comment (Andrew4Coding) [inline packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts:418]: Remove this

Comment (Andrew4Coding) [inline packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts]: remove this

**Review by Andrew4Coding — COMMENTED** [review]
remove console logs

File: packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts (hunk 1 above)
Comment (daniyusra) [reply packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts]: resolved!

**Review by Andrew4Coding — APPROVED** [review]

**Pull request merged**